concent_api/core/views.py

```diff
@@ -121,8 +121,39 @@ def receive(request, message):
 
 @api_view
 @require_POST
-def receive_out_of_band(_request):
-    return HttpResponse("out of band message sent", status = 200)
+def receive_out_of_band(request, message):
```

Comment:
pylint returns warnings for this line.

```
W:124,24: Unused argument 'request' (unused-argument)
W:124,33: Unused argument 'message' (unused-argument)
```

Please always make sure that there are no unfixed warnings before submitting a pull request.